### ISSUE TITLE
Current power consumption seems to be wrong

### ISSUE THREAD

**Issue by schroeda-hub** [Planet-Moon/SMA_pyModbus#1]
# Description

When reading current power consumption the value seems to be outside the expected range.
This maybe caused by a low update frequency.

- Example:
```
SolarPower: 244 W
LeistungEinspeisung: 4 W
AkutellerVerbrauch: 206 W (expected: SolarPower - LeistungEinspeisung => 240 W)
```

**Comment by Planet-Moon** [comment]
When reading the values from Telegram, the `SolarPower` and `LeistungEinspeisung` values are read from the device whereas `AktuellerVerbrauch` gets updated every 180 seconds. This can lead to unexpected results due to varying values.

This will be fixed to avoid further confusion.

**Comment by Planet-Moon** [comment]
Fixed by commit d8422970a82f302bbca7e42126cce430a942297a
